.travis.yml

```diff
@@ -6,16 +14,16 @@ php:
 git:
   depth: false
 script:
+  - docker-compose --version
   - test/phplint.sh
   - test/strict_types.sh
   - composer install
```

Comment:
Since we're now running PHPUnit inside the containerized PHP environment, do we still need to run `composer install`? I think you would know best, but if we don't need it then I guess we can save a little bit of time by omitting it.

Reply:
Yeah I guess since the build of the container will do this, no need for it here.